metsrw/premis_event.py

```diff
@@ -0,0 +1,40 @@
+from . import exceptions
+from . import utils
+
+
+class Event(object):
+    """
+    An object representing a PREMIS object element.
+
+    :param str event_type: Event type.
+    :param str outcome: Whether an event passed or failed.
+    """
+
+    def __init__(self, event_type=None, outcome=None):
+        self.event_type = event_type
+        self.outcome = outcome
```

Comment:
Events have more attributes than these two.  How will those be handled?  Is there the intention for adding more attributes later? [The PREMIS spec](http://www.loc.gov/standards/premis/v2/premis-2-2.pdf) is actually really readable and clear about what is in it (and optional/required, repeatable etc)